integration-cli/requirements.go

```diff
@@ -88,3 +96,14 @@ func testRequires(t *testing.T, requirements ...TestRequirement) {
 		}
 	}
 }
+
+// testRequires checks if the environment satisfies the requirements
+// for the test to run or skips the tests.
+func testRequiresWithLog(t *testing.T, msg string, requirements ...TestRequirement) {
```

Comment:
@ahmetalpbalkan The reason I do this is that the output looks cleaner (make test) and we can see Skipped tests like below...
[PASSED]: pull - pull fooblahblah1234 (non-existing image)
[SKIPPED]: pull - pull official names
[PASSED]: pull - pull images from local repo